dedisperse/dedisperse.py

```diff
@@ -1,16 +1,26 @@
 '''
-    Dedisperses data
+Dedisperses data
 '''
 # pylint: disable-msg=C0103
 import numpy as np
 
-def dedisperse(samples, dm):
+def dedisperse(samples, highest_x=None, max_delay=0, dm=None):
```

Comment:
What does DM stand for? Maybe change it to the actual meaning for example dispersion_measure so that the code stays maintainable in the future.